status-im/status-react
# Run Xcode build script against the develop branch

Fixes #2531

### Summary:

A Xcode build script was assuming the main dev branch is `master` instead of `develop`. This updates the build script to follow the project conventions. This should remove the need for jerry-rigging a `master` branch into a dev repo as well. We can update the wiki to remove this dependency.

### Steps to test:

```
git checkout master
xcodebuild -workspace StatusIm.xcworkspace -configuration Debug -scheme StatusIm -destination id=EE866A6E-0EB5-49BA-AF5D-DBBD0E9876CC -derivedDataPath build
```

You should see:

```
fatal: ambiguous argument 'master..': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
fatal: bad revision '^'
Command /bin/sh failed with exit code 128

** BUILD FAILED **
```

Next, do:

```
git remote add dbalatero https://github.com/dbalatero/status-react.git
git fetch dbalatero
git checkout "bug/xcode-build-failing-with-git-master-issue#2531"

xcodebuild -workspace StatusIm.xcworkspace -configuration Debug -scheme StatusIm -destination id=EE866A6E-0EB5-49BA-AF5D-DBBD0E9876CC -derivedDataPath build
```

You should see:

```
** BUILD SUCCEEDED **
```

status: ready
